tools/sz_repo_cli/lib/src/commands/src/deploy_web_app_command.dart

```diff
@@ -101,7 +101,8 @@ class DeployWebAppCommand extends Command {
           'build',
           'web',
           '--release',
-          '--dart-define=FLUTTER_WEB_USE_SKIA=true'
+          '--web-renderer',
+          'canvaskit',
```

Comment:
Wouldn't `auto` be better for mobile devices? Or is there sth that makes it unsuitable for us?

Reply:
We can use the canvas kit renderer for now, but back when we had the performance problems that was quite a while ago. We should test that again (might very well still be true of course).

Also what would be the overhead for "maintaining" it? I think it should more or less just work. Sure there is a slightly higher chance for bugs but if it works for us when we just scroll through once then I think the chance of new bugs occuring out of nowhere should be slim.

Reply:
I agree that our goal should be using someday `auto` but with maintaining I mean we need to (manually) test the current features but also every upcoming feature.